src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -259,6 +279,8 @@ protected function rollbackMigrations(array $migrations, $paths, array $options)
             );
         }
 
+        $this->fireMigrationEvent('afterAll:down');
```

Comment:
Think you forgot this one?

Reply:
Damnit. Its on my imac at home, so i'll be able to fix it this evening.